elastio-s3-changelog/cloudformation-multiple-buckets-with-macros.yml

```diff
@@ -14,6 +14,12 @@ Parameters:
   EnableQrts:
     Type: String
     Default: 'false'
+  QrtsBatchSize:
+    Type: Number
+    Default: 100
```

Comment:
I don't think we need to duplicate default values in these parameters. Can we just make them required?